### PR TITLE
feat(ACMS-VM-18): Admin Can Add a Vehicle to a Customer

### PR DESCRIPTION
**Pull request by MichaelVacca**
Jira ticket: https://champlainsaintlambert.atlassian.net/jira/software/projects/ACMS/boards/61/backlog?selectedIssue=ACMS-18

Changes:
    - Added the Create new vehicle functionality  
    - Also automatically updates when a vehicle is added  
    - Did all the tests for both the back-end and the front-end

### CODE REVIEW

File: acms-react-app/src/App.js [hunk 1]
```diff
@@ -41,6 +42,10 @@ const router = createBrowserRouter([
       {
         path: "/admin/customers/:customerId/vehicles",
         element: <CustomerVehicles />,
+      },
+      {
+        path: "/admin/customers/:customerId/vehicles/newVehicle",
+        element: <AddNewVehicle />,
```
Comment (CrisWillCarry) [inline acms-react-app/src/App.js:48]: it's a view... I would suggest naming it something like VehicleForm

**Review by CrisWillCarry — APPROVED** [review]
don't forget to change the PR title

File: acms-react-app/src/Pages/Admin_Pages/CustomerVehicleDetails_Page/CustomerVehicleDetails.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+    const formData = new FormData(event.target);
+    const updatedCustomerVehicle = {
+      firstName: formData.get('firstName'),
+      make: formData.get('make'),
+      model: formData.get('model'),
+      year: formData.get('year'),
+      transmissionType: formData.get('transmissionType'),
+      mileage: formData.get('mileage')
+
```
Comment (Iantomasi) [inline acms-react-app/src/Pages/Admin_Pages/CustomerVehicleDetails_Page/CustomerVehicleDetails.js:39]: Good work Michael on the create vehicle functionality Michael. Thanks for also fixing my error regarding the transmission setting in the front-end for you. Respect #Rich&Handsome

**Review by Iantomasi — DISMISSED** [review]
Good stuff bro

**Pull request closed without merging**